packages/web-components/src/components/va-search/va-search.tsx

```diff
@@ -168,6 +418,35 @@ export class VaSearch {
           <i aria-hidden="true" class="fa fa-search" />
           {buttonText && <span id="va-search-button-text">{buttonText}</span>}
         </button>
+        {isListboxVisible && (
+          <ul
+            id="va-search-listbox"
+            aria-label="Search Suggestions"
+            role="listbox"
+          >
+            {formattedSuggestions.map(
+              (suggestion: string | HTMLElement, index: number) => {
+                return (
+                  <li
+                    id={`listbox-option-${index}`}
+                    aria-selected={activeIndex === index ? 'true' : undefined}
+                    aria-hidden="true"
```

Comment:
Do we want to hide the suggestions from screen readers? I'm wondering if we could have [`aria-live="polite"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) somewhere to let the user know that there are updates to the document.